tool/cli.js

```diff
@@ -313,9 +313,24 @@ program
       const { hostname, port } = options;
       let url;
       // Perhaps they typed in a path relative to the content root
-      if (slug.startsWith("files") && slug.endsWith("index.html")) {
+      if (
+        (slug.startsWith("files") || fs.existsSync(slug)) &&
+        (slug.endsWith("index.html") || slug.endsWith("index.md"))
+      ) {
+        if (fs.existsSync(slug) && !CONTENT_TRANSLATED_ROOT) {
+          console.warn(
+            chalk.yellow(
+              `Did you forget to set the environment variable ${chalk.bold(
+                "CONTENT_TRANSLATED_ROOT"
+              )}?`
+            )
+          );
+        }
+        const slugSplit = slug.split(path.sep);
         const document = Document.read(
-          slug.split(path.sep).slice(1, -1).join(path.sep)
+          slugSplit
```

Comment:
Now it becomes more hacky. Since we have `CONTENT_ROOT`/`CONTENT_TRANSLATED_ROOT` set we can do `slug.replace(CONTENT_ROOT, "").replace(CONTENT_TRANSLATED_ROOT, "").replace(/^files/, "")` which feels a little safer. Otherwise this goes boom if a user has a folder named `files` anywhere in their hierarchy.

Reply:
I'll try to simplify it. I just get nervous about doing `.replace()` instead of proper things like `path.relative()` because you may or may not get trailing slashes that latter.